client/my-sites/posts/post-list.jsx

```diff
@@ -103,6 +104,26 @@ var Posts = React.createClass( {
 		window.removeEventListener( 'resize', this.debouncedAfterResize );
 	},
 
+	shouldComponentUpdate: function( nextProps ) {
+		const props = this.props;
```

Comment:
Not really... it's just there to avoid saying "_this_.props" all the time. But given it's just a few lines, I can remove the declaration and keep it simple.